### PR TITLE
Update welcome

### PR DESCRIPTION
**Pull request by marcelo140**
Didn't delete the "Welcome" image because I'm expecting to change back to it once codeweek is over. Am I wrong?

### CODE REVIEW

File: app/views/welcome/index.html.slim [hunk 1]
```diff
@@ -3,9 +3,11 @@
   .tp-banner
     ul
       /! SLIDE
-      li data-masterspeed="1000" data-slotamount="5" data-title=("Bem-vindo") data-transition="fade"
+      li data-masterspeed="1000" data-slotamount="5" data-title=("Codeweek") data-transition="fade"
```
Comment (martinhoaragao) [inline app/views/welcome/index.html.slim]: Can you change it to "CodeWeek"?

File: app/views/welcome/index.html.slim [hunk 2]
```diff
@@ -3,9 +3,11 @@
...
         /! MAIN IMAGE
-        = carousel_image_tag('boasvindas_v9.png')
+        = carousel_image_tag('codeweek.png')
+        = slider_link(start_second: 400, x: 1000, y: 480, title: "Ver mais",
```
Comment (martinhoaragao) [inline app/views/welcome/index.html.slim]: It needs a target _blank to open in a new tab instead. Not sure how it's done with the slider_link, I thought it had already been used as such.

Comment (marcelo140) [reply app/views/welcome/index.html.slim]: I was the one who created this helper. Fixed it already.

Comment (martinhoaragao) [reply app/views/welcome/index.html.slim]: Cool. Got it confused with another helper then. Go ahead!

**Pull request merged**